apl/tabular-operators/join-operator.mdx

```diff
@@ -110,19 +110,19 @@ LeftDataset
 
 ### Parameters
 
-- `LeftDataset`: The first dataset, also known as the outer table or the left side of the join. If you expect one of the datasets to contain consistently less data than the other, specify the smaller dataset as the left side of the join.
-- `RightDataset`: The second dataset, also known as the inner table or the right side of the join.
-- `KindOfJoin`: Optionally, the [kind of join](#kinds-of-join) to perform. The only supported value and the default is `inner`.
+- `LeftDataset`: The first dataset, also known as the outer dataset or the left side of the join. If you expect one of the datasets to contain consistently less data than the other, specify the smaller dataset as the left side of the join.
+- `RightDataset`: The second dataset, also known as the inner dataset or the right side of the join.
+- `KindOfJoin`: Optionally, the [kind of join](#kinds-of-join) to perform.
 - `Conditions`: The conditions for matching rows. The conditions determine how Axiom matches rows from the `LeftDataset` with rows from the `RightDataset`.
     - To join datasets on a field that has the same name in the two datasets, simply use the field name. For example, `on id`.
     - To join datasets on a field that has different names in the two datasets, define the two field names in an equality expression such as `on $left.id == $right.trace_id`.
     - The two sides of the equality expression must have the same data type.
-    - You can use expressions in the join conditions. For example, to compare two fields of different data types, use `on $left.id_string == tostring($right.trace_id_int)`.
+    - You can use expressions in the join conditions. For example, `on $left.status_string == tostring(500)`.
```

Comment:
Probably shouldn't have this be a constant. So it would be better to be `on $left.status_string == tostring(trace_id_int)` where `trace_id_int` is in the right dataset.